LongWarOfTheChosen/Src/LW_Overhaul/Classes/X2Condition_NotLost.uc

```diff
@@ -0,0 +1,25 @@
+
+//---------------------------------------------------------------------------------------
+//  FILE:    X2Condition_NotLost.uc
+//  AUTHOR:  Grobobobo
+//  PURPOSE: Condition that specifies that target can't be lost
+//---------------------------------------------------------------------------------------
+class X2Condition_NotLost extends X2Condition;
+
+event name CallMeetsCondition(XComGameState_BaseObject kTarget) 
+{
+    local XComGameState_Unit UnitState;
+    
+    UnitState = XComGameState_Unit(kTarget);
+    
+    if (UnitState != none)
+    {
+        if ( UnitState.GetTeam() != eTeam_TheLost )
+        {
+            return 'AA_Success'; 
+        }
+    }
+    else return 'AA_NotAUnit';
+
+    return 'AA_AbilityUnavailable';
```

Comment:
Fixed the formatting, leaving it to you if you want to remove the nested `if` and use `&&` instead:
```suggestion
	local XComGameState_Unit UnitState;

	UnitState = XComGameState_Unit(kTarget);

	if (UnitState != none)
	{
		if (UnitState.GetTeam() != eTeam_TheLost)
		{
			return 'AA_Success'; 
		}
	}
	else return 'AA_NotAUnit';

	return 'AA_AbilityUnavailable';
```